src/main/java/org/minions/devfund/abel/WhackAMole.java

```diff
@@ -125,7 +125,7 @@ public String printGridToUser() {
     public String printGrid() {
         StringBuilder stringBuilder = new StringBuilder();
         for (char[] row : this.moleGrid) {
-            stringBuilder.append(String.format("%s%s", String.valueOf(row), System.lineSeparator()));
+            stringBuilder.append(String.format("%s%n", String.valueOf(row)));
```

Comment:
I think you should only change using the following code.
```java
stringBuilder.append(row).append("\n");
```

Reply:
Windows:
![image](https://user-images.githubusercontent.com/40074674/42783918-76cfded8-891b-11e8-98d7-c05e90f9eb39.png)
Linux
![image](https://user-images.githubusercontent.com/40074674/42783944-8d2021ca-891b-11e8-9d60-8e8b72b23a92.png)

I check on both OS.
I can use "\n" but I think that is just hardcode for windows.

Reply:
Windows:
![image](https://user-images.githubusercontent.com/12613518/42784035-dd61ee84-891b-11e8-8cc9-9649c70b4c38.png)
Linux:
![image](https://user-images.githubusercontent.com/12613518/42784057-e75e0634-891b-11e8-93cf-24d3192a4f2e.png)

I can use "\n" but that is just hardcode and only works on Unix based.
https://stackoverflow.com/a/31981167